lib/fog/ovirt/compute/v4.rb

```diff
@@ -104,7 +104,7 @@ def get_attr_value(value, opts)
             when OvirtSDK4::Console
               { :enabled => value.enabled }
             else
-              if value.class.respond_to?(:parent) && value.class.parent == OvirtSDK4
+              if value.class.respond_to?(:module_parent) && value.class.module_parent == OvirtSDK4
```

Comment:
Not the intention of this PR, but you can use `try`
```suggestion
              value.class.try(:module_parent) == OvirtSDK4
```